src/web/app/maintenance-page.component.html

```diff
@@ -0,0 +1,6 @@
+<p>
+  TEAMMATES server is currently undergoing some maintenance.
```

Comment:
Maybe we could redirect them to our github page or put the support email here just in case they want to know more? e.g. what time the maintenance ends

Reply:
- GitHub page is not the way we communicate to end users
- Putting support email may be a valid idea, but how much answer can user get even if we put it there?
- Maintenance end time can't be predicted. It's also quite an overkill if that's part of application config.
- We also might want to put extra messages that really depend on case-by-case basis. It may not be a simple text field that can be fit inside a string type.

Might as well just edit this HTML file as necessary before deploying.

Reply:
Yup, let's keep it flexible. I can quickly write the message and deploy.